dicom_utils/cli/anonymize.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import shutil
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Iterable, List, Optional
+
+import pydicom
+from tqdm_multiprocessing import ConcurrentMapper
+
+from ..anonymize import anonymize
+from ..container import iterate_input_path
+from ..dicom import has_dicm_prefix
+
+
+def _anonymize(path: Path, output: Path, root: Optional[Path]) -> Path:
+    if not path.is_file():
+        raise FileNotFoundError(path)  # pragma: no cover
+
+    dest_path = output / (path.relative_to(root) if root is not None else path.name)
+    dest_path.parent.mkdir(parents=True, exist_ok=True)
+    if path.suffix.lower() == ".dcm" or has_dicm_prefix(path):
+        with pydicom.dcmread(path) as dcm:
+            anonymize(dcm)
```

Comment:
Some of this functionality already exists in `dicomphi.py`, but I don't think that CLI uses `ConcurrentMapper`. Maybe we could resolve `anonymize.py` and `dicomphi.py` into a single script?